src/crypto.rs

```diff
@@ -30,77 +31,25 @@ pub fn sha3_256(input: &[u8]) -> Digest256 {
     output
 }
 
-// Wrapper for `ed25519_dalek::Signature` that adds `Hash` and `Ord` impls.
-#[derive(Clone, Copy, Eq, PartialEq, Serialize, Deserialize)]
-pub struct Signature(pub ed25519_dalek::Signature);
-
-#[allow(clippy::derive_hash_xor_eq)]
-impl Hash for Signature {
-    fn hash<H: Hasher>(&self, state: &mut H) {
-        self.0.to_bytes().hash(state);
-    }
-}
-
-impl Ord for Signature {
-    fn cmp(&self, other: &Self) -> Ordering {
-        self.0.to_bytes().cmp(&other.0.to_bytes())
-    }
+pub fn name_from_key(public_key: &PublicKey) -> XorName {
```

Comment:
Should this not be just `name` (we can see the param is key)

Reply:
just want to make the calling to this function more explicitly.